functionary/ui/templates/base.html

```diff
@@ -31,6 +31,7 @@
               type="text/css"/>
         <title>Functionary</title>
         <script src="{% static 'js/htmx-1.8.4.min.js' %}"></script>
+        <script src="{% static 'js/hyperscript-0.9.7.min.js' %}"></script>
```

Comment:
Is this a necessity or a nice to have? This file is rather large.

Reply:
@scott-taubman is using it in his workflows fork along with the modal stuff, so I decided to go ahead and include it in this PR

Reply:
Removed hyperscript